webrtc/rtcpeerconnection/setLocalDescription.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>RTCPeerConnection setLocalDescription tests</title>
```

Comment:
Please include "ICE Gathering State" in the title too, since this test doesn't attempt to test any other aspect of setLocalDescription.